### PR TITLE
build: badge component

### PR DESCRIPTION
**Pull request by Shubhamkr2610**

### CODE REVIEW

**Review by tonymike99 — APPROVED** [review]

File: components/alert/alert.html [hunk 1]
```diff
@@ -53,11 +53,11 @@ <h3 class="aside">Getting Started</h3>
                 <ul class="aside-left-list">
                     <h3 class="aside">Components</h3>
```
Comment (itsmebipul00) [inline components/alert/alert.html]: headings shouldnt be inside a list

**Comment by itsmebipul00** [comment]
Try to use rems more

File: components/alert/alert.html (hunk 1 above)
Comment (itsmebipul00) [reply components/alert/alert.html]: only list-items inside a list

File: components/alert/alert.html [hunk 2]
```diff
@@ -35,15 +35,15 @@ <h3 class="logo-letter">E<i class="fab fa-uikit"></i>S-UI</h3>
 
 
     <!-----------------------------sidebar left starts---------------------->
-    <div class="alert-sidebar-main-container">
+    <div class="aside-sidebar-main-container">
```
Comment (arjun-computer-geek) [inline components/alert/alert.html]: I think class name is too logn?

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]

File: components/alert/alert.html (hunk 1 above)
Comment (Shubhamkr2610) [reply components/alert/alert.html]: made the changes . please review

File: components/alert/alert.html (hunk 2 above)
Comment (Shubhamkr2610) [reply components/alert/alert.html]: fixed it. Please review it

**Review by arjun-computer-geek — COMMENTED** [review]
Now looks good

**Review by arjun-computer-geek — APPROVED** [review]
It is ready to merge from my side

**Review by itsmebipul00 — APPROVED** [review]

**Pull request merged**